react-front-end/src/components/MovieDetails.js

```diff
@@ -15,15 +15,15 @@ function MovieDetails(props) {
   useEffect(() => {
     getCurrentMovieDetails();
     window.scrollTo(0, 0);
-  }, []);
+  }, [id]);
 
   useEffect(() => {
-    if(typeof currentMovieDetails !== "undefined"){
+    if (typeof currentMovieDetails !== "undefined") {
       const { fireFunction } = location.state;
-      if(fireFunction){
+      if (fireFunction) {
         handleWatchTrailerClick(currentMovieDetails.original_title);
       }
-      window.scrollTo(0,10000)
```

Comment:
This was to scroll down to the trailer when you click "watch-trailer" from the banner.

Reply:
The above code gets executed everytime user navigates to movie details page ireespective of whether user clicked on watch trailer or just to view info. To achieve your requirement, i have made code fixes to pushed the changes now.

Reply:
will resolve this for now, so i can merge with master